### PR TITLE
Added the new gsoc-alumni file

### PR DESCRIPTION
**Pull request by tharun143**
Signed-off-by: tharun143 <tharunravuri2000@gmail.com>

This the new gsoc-alumni file updated the file as per @GeorgLink suggestions.

### CODE REVIEW

**Comment by tharun143** [comment]
@GeorgLink Thank you so mcuh for your suggestions and for the handy book.

**Comment by tharun143** [comment]
@GeorgLink I committed the details of outreachy program and GSoC 2020 in this file.

File: Community/gsoc-alumni.md [hunk 1]
```diff
@@ -184,4 +344,36 @@ The Manuscripts project, which is a part of the Grimoire Toolset, helps us in an
...
+## CHAOSS Outreachy students from May 2020 to August 2020 Interns
+
+|Student|Project|Mentors|
+|-------|:-----:|-------|
+|[Ore-Aruwaji Oloruntola](#Ore-Aruwaji-Oloruntola)|Build Workflow Process for CHAOSS Diversity & Inclusion Badging|Matt Germonprez, Matt Snell, Saleh Abdel Motaal|
+
+
+### Ore-Aruwaji Oloruntola
+
+Ore-Aruwaji Oloruntola started the outreachy intern program.
```
Comment (GeorgLink) [inline Community/gsoc-alumni.md:359]: This should be parallel with the GSoC students.

```suggestion
Ore-Aruwaji Oloruntola was selected for Outreachy 2020.
```

File: Community/gsoc-alumni.md [hunk 2]
```diff
@@ -184,4 +344,36 @@ The Manuscripts project, which is a part of the Grimoire Toolset, helps us in an
...
-* [LinkedIn](https://www.linkedin.com/in/pranjalaswani/)
\ No newline at end of file
+* [LinkedIn](https://www.linkedin.com/in/pranjalaswani/)
+
+
+## CHAOSS Outreachy students from May 2020 to August 2020 Interns
+
+|Student|Project|Mentors|
+|-------|:-----:|-------|
+|[Ore-Aruwaji Oloruntola](#Ore-Aruwaji-Oloruntola)|Build Workflow Process for CHAOSS Diversity & Inclusion Badging|Matt Germonprez, Matt Snell, Saleh Abdel Motaal|
```
Comment (GeorgLink) [inline Community/gsoc-alumni.md:354]: Please move the list of Outreachy students up to the same place that the GSoC lists are. 

I would prefer to have all the lists/tables together in one place in chronological order.

File: Community/gsoc-alumni.md [hunk 3]
```diff
@@ -184,4 +344,36 @@ The Manuscripts project, which is a part of the Grimoire Toolset, helps us in an
...
+
+
+### Ore-Aruwaji Oloruntola
```
Comment (GeorgLink) [inline Community/gsoc-alumni.md:357]: I would prefer to have all students in chronological order, not separated out by program. Could you put this section please with the GSoC 2020 students?

**Pull request closed without merging**